powered-by.html

```diff
@@ -801,6 +801,11 @@
         "logo": "edenlab.svg",
         "logoBgColor": "#ffffff",
         "description": "Edenlab FHIR engineers use Kafka to manage real-time data flow between dozens of microservices in our product - <a href=https://kodjin.com>Kodjin Interoperability Suite</a> - a low-code FHIR-based infrastructure for healthcare data management (contains FHIR Server, Terminology Service, and Mapper). Edenlab is a custom software and product development company focused primarily on healthcare data interoperability. We are also working with some major customers from the Fintech space (like Mastercard ® and Raffeisen BankInternational)."
+    }, {
+	"link': "https://spitha.io/",
+	"logo": "spitha.png",
```

Comment:
The PR is missing the corresponding png file.

Reply:
I added the png file